image-classifier/webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+const commonConfig = {
+  entry: path.resolve(__dirname, './index.ts'),
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: 'ts-loader',
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js'],
+  },
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'build'),
+  },
+  plugins: [
+    new CleanWebpackPlugin(),
+    new CopyWebpackPlugin({
+      patterns: [
+        { from: 'node_modules/@mediapipe/tasks-vision/wasm', to: 'wasm' },
```

Comment:
@khanhlvg - Should we load the WASM from `unpkg` instead of having to have this rewrite?

Reply:
@schmidt-sebastian I think there're some benefits of copying the WASM from `node_modules` and self-host rather than relying on CDN:
* You only need to maintain SDK version number at one place (package.json).
* Your site doesn't go down if the CDN goes down.